src/components/app/filter-dropdown.jsx

```diff
@@ -28,8 +29,7 @@ const FilterCategory = React.createClass({
     });
   },
   onFilterInputChange() {
-    const {filterInput} = this.refs;
```

Comment:
it's probably better to use `event.target.value` instead of doing a DOM lookup (and no need for refs that way ;) )